go.mod

```diff
@@ -35,7 +35,7 @@ require (
 	github.com/prometheus/client_model v0.2.0
 	github.com/prometheus/common v0.9.1
 	github.com/satori/go.uuid v1.2.0
-	github.com/stretchr/testify v1.5.1 // indirect
+	github.com/stretchr/testify v1.5.1
```

Comment:
This change should not be there. I do not see it as an import in the change anywhere.

Reply:
you are right. I removed the usage because lint showed it as blacklisted, and forgot to remove from here as well. I will remove it